src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -241,6 +243,27 @@ export default class MutationMapperStore extends DefaultMutationMapperStore {
         return (this.mutationData.result || []).map(mutation => [mutation]);
     }
 
+    @computed get indexedAnnotatedMutationsByGenomicLocation(): {
+        [genomicLocation: string]: Mutation;
+    } {
+        if (
```

Comment:
this is a problem in many places in our codebase but we should really not be doing these checks inside of this function.    if there is not mutationData.result, this getter should not ever be called.  the loading status should be handled outside.